security-aot/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 
 micronautBuild {
     aot {
-        version.set("2.0.0-SNAPSHOT")
+        version.set("1.1.2")
```

Comment:
I was unable to resolve AOT 2.0.0-SNAPSHOT

```
cannote resolve aot 2.0.0-SNAPSHOT

Execution failed for task ':micronaut-security-aot:compileTestGroovy'.
> Could not resolve all files for configuration ':micronaut-security-aot:testCompileClasspath'.
   > Could not resolve io.micronaut.aot:micronaut-aot-core:2.0.0-SNAPSHOT.
     Required by:
         project :micronaut-security-aot
      > Unable to find a variant of io.micronaut.aot:micronaut-aot-core:2.0.0-SNAPSHOT:20230307.100234-17 providing the requested capability io.micronaut.aot:aot-core-test-fixtures:
           - Variant apiElements provides io.micronaut.aot:micronaut-aot-core:2.0.0-SNAPSHOT
           - Variant runtimeElements provides io.micronaut.aot:micronaut-aot-core:2.0.0-SNAPSHOT
           - Variant sourcesElements provides io.micronaut.aot:micronaut-aot-core:2.0.0-SNAPSHOT
           - Variant javadocElements provides io.micronaut.aot:micronaut-aot-core:2.0.0-SNAPSHOT
           - Variant testFixturesApiElements provides io.micronaut.aot:micronaut-aot-core-test-fixtures:2.0.0-SNAPSHOT and io.micronaut.aot:micronaut-micronaut-aot-core-test-fixtures:2.0.0-SNAPSHOT
           - Variant testFixturesRuntimeElements provides io.micronaut.aot:micronaut-aot-core-test-fixtures:2.0.0-SNAPSHOT and io.micronaut.aot:micronaut-micronaut-aot-core-test-fixtures:2.0.0-SNAPSHOT
           - ```

Reply:
@melix any idea?

Reply:
Yes (well, not an idea since I had to investigate). This is a side effect of moving AOT to conventional project names. Before, the AOT test fixtures were published with a wrong capability name `aot-core-test-fixtures` instead of `micronaut-aot-core-test-fixtures`.

Therefore I would have to change the build plugins to include the right capability. I will work on a fix.